<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSmsGsmModemWidget</class>
 <widget class="QWidget" name="SendSmsGsmModemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>148</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>15</number>
   </property>
   <property name="verticalSpacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDestPhone">
     <property name="text">
      <string>Dest. phone</string>
     </property>
     <property name="buddy">
      <cstring>cmbDestPhone</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cmbDestPhone">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>24</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblMsgClass">
     <property name="text">
      <string>Msg. class</string>
     </property>
     <property name="buddy">
      <cstring>cmbMsgClass</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="cmbMsgClass">
       <item>
        <property name="text">
         <string>0..Display immediately</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1..ME (mobile equipment)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2..SIM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3..TE (terminal equipment)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>24</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMsgTemplate">
     <property name="text">
      <string>Msg. template</string>
     </property>
     <property name="buddy">
      <cstring>cmbMsgTemplate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbMsgTemplate"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnMsgTemplate">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnGsmSettings">
       <property name="text">
        <string>GSM settings...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="text">
        <string>Test</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/action/resumeTest.png</normaloff>:/img/action/resumeTest.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
